### PR TITLE
Add Whispers in Nalubaale cards

### PR DESCRIPTION
**Pull request by SuperFlamingTominoHead**

### CODE REVIEW

File: pack/win.json [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        "deck_limit": 3,
+        "faction_code": "jinteki",
+        "faction_cost": 1,
+        "illustrator": "Yog Joshi",
+        "keywords": "Region",
+        "pack_code": "win",
+        "position": 16,
+        "quantity": 3,
+        "side_code": "corp",
+        "text": "Install only in the foort of HQ or R&D.\nWhenever the Runner accesses cards from this server, he or she accesses 3 additional cards. After accessing, gain 2[credit] for each card accessed. This applies even during the run on which the Runner trashes Mwanza City Grid.\nLimit 1 <strong>region</strong> per server.",
```
Comment (gereons) [inline pack/win.json]: "foort" should be "root".

File: pack/win.json [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        "cost": 0,
+        "deck_limit": 3,
+        "faction_code": "nbn",
+        "faction_cost": 2,
+        "illustrator": "Nasrul Hakim",
+        "pack_code": "win",
+        "position": 17,
+        "quantity": 3,
+        "side_code": "corp",
+        "text": "Play only if the Runner made a successful run on his or her last turn.\nName a card type, then reveal the grip. Gaint 2[credit] for each card in the grip that has the named type.",
```
Comment (gereons) [inline pack/win.json]: "Gaint" should be "gain"

**Comment by Alsciende** [comment]
@SuperFlamingTominoHead Freedom Khumalo: Crypto Anarchist is missing a base link

**Comment by gereons** [comment]
added in #299

**Pull request merged**